### DISCUSSION TITLE
1993-07-01-gou-ppt

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#115] [General]
# 大环境不好的情况下如何🐶着 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life大概是今年4月份给团队同学的一个分享，归档于博客，来聊聊作为一个国内的开发工程师，如何在大环境不好的情况🐶着变强。

https://tw93.fun/2022-07-01/gou.html

**Comment by chenxofhit** [comment]
这个文件有源文件吗？

**Reply by tw93** [reply]
有的 晚点上传一下

**Reply by tw93** [reply]
https://tw93.fun/images/pdf/%E5%A4%A7%E7%8E%AF%E5%A2%83%E4%B8%8D%E5%A5%BD%E6%83%85%E5%86%B5%E4%B8%8B%E5%A6%82%E4%BD%95%F0%9F%90%B6%E7%9D%80_220429_Tw93.pdf

**Comment by ZhongFuze** [comment]
谢谢, 这篇文章, 今年依旧有被安慰到, 🐶着变强
